client/signup/steps/user/index.jsx

```diff
@@ -51,14 +54,33 @@ export class UserStep extends Component {
 	}
 
 	setSubHeaderText( props ) {
+		const { flowName, oauth2Client, translate } = props;
+
 		let subHeaderText = props.subHeaderText;
 
-		if ( props.flowName === 'social' ) {
+		if ( flowName === 'social' ) {
 			// Hides sub header for this particular flow
 			subHeaderText = '';
-		} else if ( 1 === signupUtils.getFlowSteps( props.flowName ).length ) {
+		} else if ( flowName === 'wpcc' && oauth2Client ) {
+			if ( oauth2Client.id === 50916 ) {
```

Comment:
In my client, `oauth2Client.id === "50916"`. Due to this, it shows the incorrect subtitle text, as seen in this screenshot:

![create_a_site_ _wordpress_com](https://user-images.githubusercontent.com/4044428/29886372-14127a1c-8d77-11e7-8aff-0db5a9cc3aab.png)

This seems to occur when I navigate from the login to the signup page via clicking a link. When I open the signup URL directly in a new tab, the correct subtitle text appears.

Reply:
Good catch! I'm going to normalize the data returned by the API endpoint to get rid of that issue.